src/api/status/src/server.js

```diff
@@ -54,5 +69,39 @@ service.router.get(`${process.env.PATH_PREFIX || ''}/status`, (req, res) => {
     .catch((err) => res.status(500).json({ error: err.message }));
 });
 
+// Get Home page
+service.router.get(process.env.PATH_PREFIX || '/', async (req, res) => {
+  const [telescope, satellite, totalPost, totalFeeds] = await Promise.all([
```

Comment:
`try/catch` here, since this can all fail, and will crash the server if you don't handle the promise rejection.

Reply:
Noted will change